README.md

```diff
@@ -37,8 +37,11 @@ When using Grape with Swagger via [grape-swagger](https://github.com/ruby-grape/
 
 ```ruby
 # FastJsonapi serializer example
+
+# app/serializers/base_serializer.rb
+class BaseSerializer; end
```

Comment:
If we are going  to encourage using a wrapper in the docs, should we `include FastJsonapi::ObjectSerializer` in this wrapper, instead of including it in the inherited classes?

Reply:
or maybe just revert back to the old example and leave an explanation?  I guess it would be a contrived, since it's not that realistic where you would only want a single serializer support 

I made this mistake when testing

```ruby
GrapeSwagger.model_parsers.register(GrapeSwagger::FastJsonapi::Parser, UserSerializer)
GrapeSwagger.model_parsers.register(GrapeSwagger::FastJsonapi::Parser, ActivitySerializer)
```

I didn't even realize what register was actually doing until i saw the`Roar::Decorator`, `Representable::Decorator` examples

Reply:
I guess reverting back with an explanation is more relevant